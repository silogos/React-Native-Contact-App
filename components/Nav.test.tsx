import React from "react";
import { render } from "@testing-library/react-native";

import NavBar from "./Nav";

describe("Nav", () => {
  it("renders correctly", () => {
    const elm = render(<NavBar />).toJSON();

    expect(elm).toMatchSnapshot();
  });

  it("Render with title", () => {
    const title = "Test Title";
    const { getByText } = render(<NavBar title={title} />);
    const titleElement = getByText(title);

    expect(titleElement).toBeTruthy();
  });
});
