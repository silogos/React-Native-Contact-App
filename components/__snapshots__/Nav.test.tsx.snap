// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders correctly 1`] = `
<View
  id="nav"
  style={
    {
      "alignItems": "stretch",
      "flexDirection": "row",
      "height": 52,
      "justifyContent": "space-between",
      "position": "relative",
    }
  }
>
  <View
    style={
      {
        "alignItems": "stretch",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      accessibilityState={
        {
          "busy": undefined,
          "checked": undefined,
          "disabled": undefined,
          "expanded": undefined,
          "selected": undefined,
        }
      }
      accessibilityValue={
        {
          "max": undefined,
          "min": undefined,
          "now": undefined,
          "text": undefined,
        }
      }
      accessible={true}
      collapsable={false}
      focusable={true}
      href="../"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      role="link"
      style={
        {
          "justifyContent": "center",
          "paddingHorizontal": 8,
        }
      }
    >
      <Text />
    </View>
  </View>
  <View
    style={
      {
        "alignItems": "stretch",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  />
</View>
`;
